lib/tessel/version.js

```diff
@@ -0,0 +1,19 @@
+// System Objects
+// ...
+
+// Third Party Dependencies
+// ...
+
+// Internal
+var Tessel = require('./tessel');
+
+/*
+  Gathers node version.
+*/
+Tessel.prototype.fetchCurrentNodeVersion = function() {
+  return this.simpleExec(['node', '--version'])
+    .then(function versionRead(version) {
```

Comment:
I know.. :smile:. Going forward we want to avoid adding more, and cleaning up as we go.